antora/ec-policies-antora-extension/index.js

```diff
@@ -6,6 +6,18 @@ const opa = import("@zregvart/opa-inspect")
 // Helpers for handlebars templates
 const hbsHelpers = {
 
+  // Determines if value is in array
+  isInArray: (value, array) => {
+    if (Array.isArray(array)) {
+      return array.includes(value);
+    } else if (typeof array === 'string') {
```

Comment:
This is just a random idea, not a blocker for the PR. I was looking at https://stackoverflow.com/questions/18903003/passing-arbitrary-arguments-to-handlebars-helpers and now I'm doubtful that it will be possible.